src/NzbDrone.Core/Tv/RefreshEpisodeService.cs

```diff
@@ -34,6 +34,7 @@ public void RefreshEpisodeInfo(Series series, IEnumerable<Episode> remoteEpisode
 
             var existingEpisodes = _episodeService.GetEpisodeBySeries(series.Id);
             var seasons = series.Seasons;
+            var hasExisting = existingEpisodes.Any();
```

Comment:
Will this be a problem for new series that didn't initially have episodes? It's rare with TVDB, but other metadata providers could handle that differently.

Perhaps we should treat season 1 differently? I think the worst case there is it is older than 14 days and gets unmonitored anyways.

Reply:
valid point, i was going to say that new episodes air in the future.
How about: If any new episode aired < 14. We unmonitor all new episodes unless they air today/future?
Should cover both cases, new series get monitored properly unless they were seriously behind with adding it. And any se

Reply:
Make it 1 day to deal with the TVDB caching/delays in being added for whatever reason? Otherwise I think that sounds good.